### PR TITLE
Attribution for SVG creative commons Map resource

### LINKED ISSUE #2868: Add Attribution content to site to comply with creative commons requirements?
We're using a Creative Commons map svg https://commons.wikimedia.org/wiki/File:Blank_USA,_w_territories.svg and need to give attribution somewhere. 

Do we have such a page yet? Seems like we should. 

Also, there's a comment in the .svg file (which is just text) giving attribution (where the svg came from), but I'm not sure if this is enough.

use title or alt for the attribution

### PR DESCRIPTION
**Pull request by kevinhaube**
This PR adds an attribution to the correct license under the Map svg we use on the Where We're Live and Home pages.

Test Steps:
1. Hover over the map on the Home page to view the cc attribution
2. Navigate to Product > Where We're Live and view the cc attribution with links under the map

## Changes
- Adds map attributions

Screen Shot 2022-12-19 at 10 02 07 AM

Screen Shot 2022-12-19 at 9 42 48 AM

### Testing
- [ ] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [X] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Linked Issues
- Fixes #2868 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### CODE REVIEW

File: frontend-react/src/pages/product/legacy-page-content/WhereWereLive.tsx [hunk 1]
```diff
@@ -26,8 +26,21 @@ export const WhereWereLive = () => {
                 src={usamapsvg}
                 alt="Map of states using ReportStream"
             />
-            ReportStream has established connections to send and report public
-            health data for each of the states and territories listed here.
+            <p className="font-body-2xs text-gray-50">
+                <a
```
Comment (stephenkao) [inline frontend-react/src/pages/product/legacy-page-content/WhereWereLive.tsx:30]: VERY superficial point completely out of curiosity, but do we have a general policy of opening external links in a new tab or keeping it on the same tab?

**Review by stephenkao — APPROVED** [review]
Looks good!

File: frontend-react/src/pages/product/legacy-page-content/WhereWereLive.tsx (hunk 1 above)
Comment (kevinhaube) [reply frontend-react/src/pages/product/legacy-page-content/WhereWereLive.tsx:30]: Great question! So as of `2.11` they removed their official guidance on the matter due to confusion from user research over the icon itself meaning "share", potentially. However, no guidance on the page now indicates _not_ to do it, which tells me it's more of an "us" call!

https://designsystem.digital.gov/components/link/ (scroll to bottom)

Screen Shot 2022-12-19 at 5 19 00 PM

Comment (stephenkao) [reply frontend-react/src/pages/product/legacy-page-content/WhereWereLive.tsx:30]: Makes sense!  Thanks for following up!

**Pull request merged**